next.config.mjs

```diff
@@ -38,6 +38,16 @@ const nextConfig = {
         hostname: getAllowedHostname(),
         pathname: "/**",
       },
+      {
+        protocol: "https",
+        hostname: "fbiinfo-present.dbc.dk",
+        pathname: "/**",
+      },
+      {
+        protocol: "https",
+        hostname: "default-forsider.dbc.dk",
+        pathname: "/**",
+      },
```

Comment:
Just curious: Why is it necessary to restrict image source? What are the security implications by not having the restrictions?